test/e2e/node/kubelet.go

```diff
@@ -479,62 +485,266 @@ var _ = SIGDescribe("kubelet", func() {
 
 		/*
 			Test if kubectl node-logs <node-name>
-			returns something or not!
+			returns an error or not!
 		*/
 
-		ginkgo.It("should return the logs ", func() {
+		ginkgo.It("should return the error of running without --query option", func() {
 			ginkgo.By("Starting the command")
 			tk := e2ekubectl.NewTestKubeconfig(framework.TestContext.CertDir, framework.TestContext.Host, framework.TestContext.KubeConfig, framework.TestContext.KubeContext, framework.TestContext.KubectlPath, ns)
 
 			for nodeName := range nodeNames {
 				cmd := tk.KubectlCmd("alpha", "node-logs", nodeName)
-				runKubectlCommand(cmd, "")
+				_, _, err := framework.StartCmdAndStreamOutput(cmd)
+				if err != nil {
+					framework.Failf("Failed to start kubectl command: %v", err)
+				}
+				err = cmd.Wait()
+				if err == nil {
+					framework.Failf("Command failed: %v. It was expected an error!", cmd)
```

Comment:
I wonder what `%v` will output for `*exec.Cmd` type. Not sure it will be readable. Can you show the log example?

Reply:
The output will be the command ran: `Command failed: /usr/bin/kubectl --server=... --kubeconfig=/etc/kubernetes/admin.conf --namespace=kubelet-9887 alpha node-logs <node-name>